bundles/org.openhab.binding.omnilink/README.md

```diff
@@ -88,7 +87,7 @@ The devices support some of the following channels:
 | `zone_volume`               | Dimmer               | Volume level of this audio zone.                                                     | `audio_zone`                                        |
 | `zone_source`               | Number               | Source for this audio zone.                                                          | `audio_zone`                                        |
 | `zone_control`              | Player               | Control the audio zone, e.g. start/stop/next/previous.                               | `audio_zone`                                        |
-| `system_date`               | DateTime             | Set controller date/time.                                                            | `controller`                                        |
+| `system_date`               | DateTime             | Controller date/time.                                                            | `controller`                                        |
```

Comment:
You could also refer/link to the rule actions chapter here for instruction how to set system date/time.